### PR TITLE
Rename "proportionality to" to "ratio"

### LINKED ISSUE #555: Make "ratio" the primary label for PATO:0001470 ! proportionality to
```yaml
id: PATO:0001470
name: proportionality to
def: "A quality inhering in a bearer by virtue of the bearer's magnitude in respect to a related entity." [PATOC:nw]
synonym: "proportion" EXACT []
synonym: "proportionality" EXACT []
synonym: "quotient" EXACT []
synonym: "ratio" EXACT [] !! **make this the primary label**
synonym: "rate" RELATED []
is_a: PATO:0001018 ! physical quality
```

### PR DESCRIPTION
**Pull request by rays22**
of
* PATO:0001470 ratio and child terms
* PATO:0040043 increased ratio
* PATO:0040042 decreased ratio

Also, improve the textual definitions and
improve the logical definitions of 'increased ratio' and 'decreased' by using EQs.

Resolve #555

### CODE REVIEW

**Review by matentzn — CHANGES_REQUESTED** [review]
I agree with the general changes at first glance but this needs @sbello @cmungall review.. (especially the renaming bit). 

I guess it comes down to the question if the new "proportionality to" is really the same in spirit as "ratio" - I hope so :)

**Comment by sbello** [comment]
I see proportionality to used in the eqs for 4 MP terms (eg.abnormal lymph node cell ratio) these have eqs in the form
'has part' some 
('proportionality to' and ('characteristic of' some 'X') and (towards some Y) and ('has modifier' some abnormal))
I think ratio slots into this just fine. Most of the terms use ratio in the label already.

We aren't currently using increased or decreased child terms

The one thing that gives me pause is this part of the comment on the old term "Not all ratios measure proportionality, eg m/s measures velocity which is not a proportion in the sense defined above, it is a quality in its own right." But looking at the definition in both the old and new versions, I think the inclusion of 'the magnitude of another entity' excludes velocity since this involves a single entity.

I think the changes to the definition do not significantly alter the meaning of the term, so I am okay with this change.

**Review by sbello — DISMISSED** [review]
I think this change is acceptable (longer comment in conversation)

File: src/ontology/pato-edit.obo [hunk 1]
```diff
@@ -20661,28 +20661,30 @@ property_value: http://purl.org/dc/terms/contributor https://orcid.org/0000-0001
 
 [Term]
 id: PATO:0040042
-name: decreased proportionality to
-def: "A proportionality quality inhering in a bearer by virtue of the bearer's decreased magnitude in respect to a related entity." [https://orcid.org/0000-0001-5208-3432]
+name: decreased ratio
+def: "A decreased quality inhering in a bearer by virtue of the bearer's decreased magnitude in proportion to the magnitude of anoter entity." [] {http://purl.org/dc/terms/contributor="https://orcid.org/0000-0001-8314-2140", http://purl.org/dc/terms/contributor="https://orcid.org/0000-0001-5208-3432"}
```
Comment (balhoff) [inline src/ontology/pato-edit.obo:20665]:
```suggestion
def: "A decreased quality inhering in a bearer by virtue of the bearer's decreased magnitude in proportion to the magnitude of another entity." [] {http://purl.org/dc/terms/contributor="https://orcid.org/0000-0001-8314-2140", http://purl.org/dc/terms/contributor="https://orcid.org/0000-0001-5208-3432"}
```

Comment (balhoff) [reply src/ontology/pato-edit.obo:20665]: Overall looks good, just a typo here.

Comment (rays22) [reply src/ontology/pato-edit.obo:20665]: Thanks @balhoff for catching it!

**Review by matentzn — APPROVED** [review]

**Comment by ar-ibrahim** [comment]
@rays22 Looks good. I just have a comment on whether the synonyms in the pattern should also reflect 'percentage' terms. For example terms like  [EFO:0007993 lymphocyte percentage of leukocytes](https://www.ebi.ac.uk/ols4/ontologies/efo/classes/http%253A%252F%252Fwww.ebi.ac.uk%252Fefo%252FEFO_0007993) describe a ratio relationship.

**Comment by rays22** [comment]
> @rays22 Looks good. I just have a comment on whether the synonyms in the pattern should also reflect 'percentage' terms. For example terms like [EFO:0007993 lymphocyte percentage of leukocytes](https://www.ebi.ac.uk/ols4/ontologies/efo/classes/http%253A%252F%252Fwww.ebi.ac.uk%252Fefo%252FEFO_0007993) describe a ratio relationship.

@ar-ibrahim , In the specific case of EFO:0007993, yes, using the PATIO 'ratio' quality would be appropriate based on this part of the textual definition: [... presented as a ratio, fraction, quotient or percentage.](https://www.ebi.ac.uk/ols4/ontologies/efo/classes/http%253A%252F%252Fwww.ebi.ac.uk%252Fefo%252FEFO_0007993)

**Review by balhoff — APPROVED** [review]

**Pull request merged**